scripts/clear-signed-agreements.py

```diff
@@ -87,4 +92,10 @@ def match_signed_agreements(supplier_ids, path):
 
```

Comment:
Line 91 is still looking for the `supplier_ids` parameter we've gotten rid of (which, presumably was a comma-separated list).

```
# 91 
supplier_ids = arguments['--supplier_ids']
```

I think we just want it gone.